module/TaskManagement/src/TaskManagement/Service/EventSourcingTaskService.php

```diff
@@ -59,34 +62,77 @@ public function findTasks(Organization $organization, $offset, $limit)
 			->orderBy('t.mostRecentEditAt', 'DESC')
 			->setFirstResult($offset)
 			->setMaxResults($limit)
-			->setParameter(':organization', $organization)
-			->getQuery();
-		return $query->getResult();
+			->setParameter(':organization', $organization);
+
+		if($startOn !== null){
+			$query->andWhere('t.createdAt >= :startOn')
+				->setParameter('startOn', $startOn->format("Y-m-d")." 00:00:00");

```

Comment:
Ciò che riceve in ingresso è un DateTime, quindi anche ore minuti e secondi sono significativi. Devi rispettare quello che viene passato. Semmai la configurazione di 00:00:00 o 23:59:59 deve avvenire nel controller quando si definisce quanto vale startOn e endOn